practice/00-HTML-CSS-basics/amultimediafile.html

```diff
@@ -0,0 +1,16 @@
+ <!DOCTYPE html>
+<html>
+	<head>
+  		<title>Don't worry</title> 
+   	</head>
+   	<body> 
+   		<h2>Don't worry</h2>
+   		<audio controls>  	
+		   	<h3>Don't worry</h3>
+		   	<source src="behappy.ogg" type="audio/ogg">
+	   </audio>
+	   <h2>Be happy</h2>
+	   <p><iframe width="250" height="185" src="https://www.youtube.com/embed/pBMKpWVKWnI" frameborder="0" allowfullscreen></iframe></p>
```

Comment:
ok (y)

2015-02-14 1:00 GMT-03:00 snunezglobant <notifications@github.com>:

> In practice/00-HTML-CSS-basics/amultimediafile.html
> <https://github.com/snunezglobant/html5bootcamp/pull/14#discussion_r24711898>
> :
>
> > @@ -0,0 +1,16 @@
> > + <!DOCTYPE html>
> > +<html>
> > +	<head>
> > +  		<title>Don't worry</title>
> > +   	</head>
> > +   	<body>
> > +   		<h2>Don't worry</h2>
> > +   		<audio controls>  	
> > +		   	<h3>Don't worry</h3>
> > +		   	<source src="behappy.ogg" type="audio/ogg">
> > +	   </audio>
> > +	   <h2>Be happy</h2>
> > +	   <p><iframe width="250" height="185" src="https://www.youtube.com/embed/pBMKpWVKWnI" frameborder="0" allowfullscreen></iframe></p>
>
> Please, you should use the tag html5 instead of an iframe. Thanks!
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/snunezglobant/html5bootcamp/pull/14/files#r24711898>.
>